lib/Net/DRI.pm

```diff
@@ -201,7 +201,6 @@ sub new
      'AU' => 'auDA',
      'BE' => 'DNSBelgium',
      'BH' => 'TRA::BH',
-     'BIZ' => 'Neustar::BIZ',
```

Comment:
This should be changed to => Neustar::NARWHAL , and an extra line should be added
Neustar::BIZ => Neustar::NARWHAL
This is added for backwards compatibility for client code that is not updated.